content/en/blog/2024/kubecon-china-2024.md

```diff
@@ -0,0 +1,56 @@
+---
+title:
+  OpenTelemetry Talks at Kubecon + CloudNativeCon + Open Source Summit + AI_dev
+  China 2024
+linkTitle: Kubecon China 2024
+date: 2024-06-25
+# prettier-ignore
+cSpell:ignore: Alhamdani Censhare EBPF Hrabusa Husni Huxing Jiahang Krom Sianturi Wanqi Zhang Zihao Ziyi
+author: '[Tiffany Hrabusa](https://github.com/tiffany76)'
+draft: true # TODO: remove this line once your post is ready to be published
+sig: Communications SIG
+---
+
+Join the OpenTelemetry project maintainers and members of the governance and
+technical committees at the [KubeCon + CloudNativeCon + Open Source Summit
++ AI_dev] conference in Hong Kong from August 21 to 23, 2024.
+
+Read on to learn about all the OpenTelemetry events at KubeCon China 2024.
+
+This post will be updated as we receive notice of other activities. Check back
+before the conference begins!
+
+## AI_dev talks
+
+- **[Adaptive Tracing Propagation with OpenTelemetry: Navigating Protocol Diversity in the Cloud | Project Lightning Talk](https://sched.co/1f4zX)**<br>
+  Wednesday, August 21 • 12:03 - 12:08
+- **[Telemetry API and OpenTelemetry: the answer of istio monitoring? | Project Lightning Talk](https://sched.co/1f4zz)**<br>
+  Wednesday, August 21 • 12:24 - 12:29
```

Comment:
do we know who the speaker of these are? The "Project Lightning Talks" are official project opportunities, so I'd like to understand who is doing them, cc @open-telemetry/governance-committee @open-telemetry/technical-committee